templates/include/telemetry/coreCommon.1.schema.json

```diff
@@ -6,7 +6,8 @@
   "type": "string"
 },
 "os": {
-  "type": "string"
+  "type": "string",
+  "enum": [ "Android", "iOS", "iPhone OS" ]
```

Comment:
we need to validate `os` if we are using it as a partition, this is a temporary solution until [Bug 1455381](https://bugzilla.mozilla.org/show_bug.cgi?id=1455381) is implemented

Reply:
Instead of dropping these entirely when os isn't an expected value I think we should probably just change the parquet output message matcher to match only the specific os values we care about (for now).